schemas/dbt_yml_files.json

```diff
@@ -712,10 +767,79 @@
         },
         "additionalProperties": false
       },
+      "include_exclude": {
+        "type": "object",
+        "properties": {
+          "include": {
+            "$ref": "#/$defs/string_or_array_of_strings"
+          },
+          "exclude": {
+            "$ref": "#/$defs/string_or_array_of_strings"
+          }
+        }
+      },
       "jinja_string": {
         "type": "string",
         "pattern": "\\{\\{.*\\}\\}"
       },
+      "model_configs": {
+        "type": "object",
+        "properties": {
+          "contract": {
+            "type": "object",
+            "properties": {
+              "enforced": {
+                "$ref": "#/$defs/boolean_or_jinja_string"
+              }
+            }
+          },
+          "grant_access_to": {
+            "title": "Authorized views",
+            "type": "array",
+            "description": "Configuration, specific to BigQuery adapter, used to setup authorized views.",
+            "items": {
+              "type": "object",
+              "required": ["database", "project"],
+              "properties": {
+                "database": {
+                  "type": "string"
+                },
+                "project": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            }
+          },
+          "hours_to_expiration": {
+            "type": "number",
+            "description": "Configuration specific to BigQuery adapter used to set an expiration delay (in hours) to a table."
+          },
+          "kms_key_name": {
+            "type": "string",
+            "description": "Configuration of the KMS key name, specific to BigQuery adapter.",
+            "pattern": "projects/[a-zA-Z0-9_-]*/locations/[a-zA-Z0-9_-]*/keyRings/.*/cryptoKeys/.*"
+          },
+          "labels": {
+            "title": "Label configs",
+            "type": "object",
+            "description": "Configuration specific to BigQuery adapter used to add labels and tags to tables/views created by dbt.",
+            "patternProperties": {
+              "^[a-z][a-z0-9_-]{0,62}$": {
+                "type": "string",
+                "pattern": "^[a-z0-9_-]{0,63}$"
+              }
+            },
+            "additionalProperties": false
+          },
+          "materialized": {
+            "type": "string"
+          },
+          "sql_header": {
+            "type": "string"
+          }
+        }
+      },
```

Comment:
I'm assuming that you didn’t' chage anything here and it's just moving it into its own def block for reuse purposes, so I haven't reviewed this